### PR TITLE
new header and file to embed

### PR DESCRIPTION
**Pull request by rebeccakay**

### CODE REVIEW

**Comment by nigef** [comment]
It appears there are changes to the template and styles since your last merge. 
If @david-yz-liu agrees, please merge from master (and fix, if any conflicts) first.

**Comment by nigef** [comment]
Use a new stylesheet rather than pasting the vendor (mdl) styles into ours.

File: python_ta/reporters/templates/template.html [hunk 1]
```diff
@@ -4,111 +4,110 @@
...
-                {% endfor %}
+                    <a href="mailto:david@cs.toronto.edu">
+                        <div id="report_bug">
+                            Found a bug? Report it to Prof. Liu!
+                        </div>
+                    </a>
+                    <!--<script src="https://code.jquery.com/jquery-3.2.1.min.js"
+                        integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+                        crossorigin="anonymous"></script>-->
+                    <script src="python_ta\reporters\templates\jquery-3.2.1.min.js"></script>
```
Comment (nigef) [inline python_ta/reporters/templates/template.html]: Don't add changes that already exist in master. This applies to many other places as well.

Comment (rebeccakay) [inline python_ta/reporters/templates/template.html]: I fixed the conflicts but the header won't stay fixed?

File: python_ta/reporters/templates/stylesheet.css [hunk 2]
```diff
@@ -1,19 +1,15 @@
-body {
```
Comment (nigef) [inline python_ta/reporters/templates/stylesheet.css]: Don't change this. You can add a new class somewhere else.

**Comment by nigef** [comment]
Redo template.html starting with a fresh copy from master.
There are too many outdated things in your version of template.html

File: python_ta/reporters/templates/stylesheet.css (hunk 2 above)
Comment (rebeccakay) [inline python_ta/reporters/templates/stylesheet.css]: I changed it because it doesn't recognize the new body

Comment (nigef) [inline python_ta/reporters/templates/stylesheet.css]: Please leave the body. You can add an additional class if needed.

Regarding the outdated, my mistake, it looks like it has not been merged, https://github.com/pyta-uoft/pyta/pull/304

Comment (rebeccakay) [inline python_ta/reporters/templates/stylesheet.css]: The header issue is still there though.

**Pull request closed without merging**